#!/usr/bin/python

SUCCESS=0
ERROR=1

x=1


assert(x == SUCCESS), "success"
